<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>17</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(41, 159, 156);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="total_time_game_screen">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 122, 118);
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>TOTAL TIME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="level_game_screen">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 122, 118);
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>LEVEL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="level_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>150</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>90</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: #037a76;
border-color: #037a76;
border-style: solid;
border-width:2px;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>TIME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="flash_card_game_screen">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: #037a76;
background-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_game_screen">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>690</y>
      <width>420</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>420</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: #037a76;
border-color: #037a76;
border-style: solid;
border-width:2px;
border-radius:10px;</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="false_game_screen">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>100</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>150</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: white;
color: rgb(255, 255, 255);
background-color: rgb(237,27,118);
border-style: solid;
border-width:2px;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>FALSE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="true_game_screen">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>260</y>
      <width>100</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>150</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 122, 118);
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>TRUE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="num_of_true_game_screen">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>320</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;
color: rgb(3, 122, 118);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="num_of_false_game_screen">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>320</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;
color: rgb(237,27,118);
background-color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;
color: rgb(3, 122, 118);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="language_flash_card_game_screen">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: rgb(3, 122, 118);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Netherlands</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="language_flash_card_game_screen_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: rgb(3, 122, 118);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>je</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_login_page">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>690</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(237, 27, 118);

color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:10px;
}

QPushButton:hover{
background-color: rgb(255, 255, 255);
color: #f44786;
border-color: #f44786;
border-width:3px;
}</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_login_page">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>690</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(237, 27, 118);

color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:10px;
}

QPushButton:hover{
background-color: rgb(255, 255, 255);
color: #f44786;
border-color: #f44786;
border-width:3px;
}</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="false_button_game_screen">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(237, 27, 118);

color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:3px;
border-radius:50px;
}

QPushButton:hover{
background-color: rgb(255, 255, 255);
color: #f44786;
border-color: #f44786;
border-width:3px;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="true_button_game_screen">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>530</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:50px;
background-color: #037a76;
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:4px;
}

QPushButton:hover{
background-color: rgb(255, 255, 255);
color: #037a76;;
border-color: #037a76;
border-width:3px;
}</string>
    </property>
    <property name="text">
     <string>âœ“</string>
    </property>
   </widget>
   <zorder>level_2</zorder>
   <zorder>label_5</zorder>
   <zorder>total_time_game_screen</zorder>
   <zorder>level_game_screen</zorder>
   <zorder>flash_card_game_screen</zorder>
   <zorder>progress_game_screen</zorder>
   <zorder>false_game_screen</zorder>
   <zorder>true_game_screen</zorder>
   <zorder>num_of_true_game_screen</zorder>
   <zorder>num_of_false_game_screen</zorder>
   <zorder>language_flash_card_game_screen</zorder>
   <zorder>language_flash_card_game_screen_2</zorder>
   <zorder>exit_login_page</zorder>
   <zorder>back_login_page</zorder>
   <zorder>false_button_game_screen</zorder>
   <zorder>true_button_game_screen</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
